packages/react-devtools-shared/src/__tests__/inspectedElementContext-test.js

```diff
@@ -521,6 +521,12 @@ describe('InspectedElementContext', () => {
     const Example = () => null;
 
     const arrayOfArrays = [[['abc', 123, true], []]];
+
+    // Todo: Uncomment if this feature is implemented https://github.com/jsdom/jsdom/issues/1892
+    // const iframe = document.createElement('iframe');
+    // iframe.setAttribute('sandbox', '');
+    // window.document.body.appendChild(iframe);
```

Comment:
Can we add something like this as [a fixture test](https://github.com/facebook/react/tree/master/fixtures/devtools) in the meanwhile? (It doesn't seem too likely that JSDom will implement this.)

Reply:
Ok I will add the test :)